capstone/src/main/webapp/youtubeScript.js

```diff
@@ -264,9 +278,42 @@ function showCategories() {
     radiobox.onclick = function() {
       enableTextInput(this);   
     }
-    const container = document.getElementById('category-container');
-    container.appendChild(radiobox);
-    container.appendChild(label);
-    container.appendChild(document.createTextNode (" "));
+    const categoryContainer = document.getElementById('category-container');
+    categoryContainer.appendChild(radiobox);
+    categoryContainer.appendChild(label);
+    categoryContainer.appendChild(document.createTextNode(" "));
+  }
+}
+
+/** Converts perspective results to knoop scale then to mohs*/
+function perspectiveToxicityScale(attributeAverages) {
```

Comment:
small nit: it might be good to create 2 separate functions, one for calculating the score using mohs (which wouldn't update the UI at all), and another function that set's the value on the UI.  Then you can use slightly more clear names, maybe like `getScoreInMohs` and `showPerspectiveToxicityScale`.  I think that is a little more clear than `perspectiveToxicityScale` which doesn't really have any verbs to explain what is done.

Reply:
I've made this change in "clarify perspectiveToxicityScale"